psycop/common/model_training_v2/trainer/preprocessing/steps/column_validator.py

```diff
@@ -76,7 +76,7 @@ def from_list(
         return cls(prefix=prefix, count=count)  # type: ignore
 
 
-@BaselineRegistry.preprocessing.register("prefix_count_validator")
+@BaselineRegistry.preprocessing.register("column_prefix_count_expectation")
```

Comment:
@HLasse I didn't get an error that 'prefix_count_validator' no longer exists in the registries. Perhaps we want the generated placeholders to include addressing? E.g.:

```cfg
[placeholder]
@preprocessing = "prefix_count_validator"
```